tubular-react-common/index.md

```diff
@@ -161,15 +270,65 @@ description: "Check how to improve your CRUD views with amazing webcontrols from
           Object - A Tubular state with all the tubular properties.
           Function - A Tubular instance conformer that returns a set of functions to execute over your source data.
         </p>
+    </div>
+```tsx
+{% raw %}
+import React from "react";
+import "./styles.css";
```

Comment:
Remove, remember FOCUS in the actual code not in implementation details. The functional code is in the CodeSandbox